content/en/docs/reference/issues-security/official-cve-feed.md

```diff
@@ -7,34 +7,24 @@ outputs:
 layout: cve-feed
 ---
 
-{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
```

Comment:
This is unrelated to the PR but are you sure we should completely remove this?

Reply:
With the current implementation we have to remove this shortcode to make it work how I want, because shortcode indices are for _all_ kinds of shortcode.

Also, though, the feed isn't particularly tied to any Kubernetes release. We could mark is as alpha in the page text though.

Reply:
Ah ok, it might be an issue since I'm supposed to merge another shortcode with the tabs thing on the RSS/JSON feed thing.